docs/cli_usage_guide.md

```diff
@@ -0,0 +1,149 @@
+- `jira_project`: The Jira project you'd like the issue to be filed under.
+- `ignore`[OPTIONAL]: A value that be set to "true" or "false" and allows the user to define `step`/`failure_type` combinations that should be ignored when creating tickets.
+
+The firewatch configuration can be saved to a file (can be stored wherever you want and named whatever you want, it must be JSON though) or defined in the `FIREWATCH_CONFIG` variable. When using the [`report` command](#report---create-jira-issues), if an argument for `--firewatch_config_path` is not provided, the environment variable will be used.
+
+## Usage
+
+### `report`
+
+The `report` command is used to generate and file Jira issues for a failed OpenShift CI run using a [user-defined firewatch configuration](#configuration).
+Many of the arguments for this command have set defaults or will use an environment variable
+
+**Pre-requisites:**
+
+1. A Jira configuration file must exist. Use the `jira_config_gen` command to generate the configuration file.
+2. A firewatch config must be defined. Use the [Configuration section above](#configuration) to generate your configuration.
+
+**Arguments:**
+
+```commandline
+Usage: firewatch report [OPTIONS]
+
+Options:
+  --fail_with_test_failures     Firewatch will fail with a non-zero exit code
+                                if a test failure is found.
+  --jira_config_path TEXT       The path to the jira configuration file
+                                [required]
+  --firewatch_config_path TEXT  The path to the firewatch configuration file
+  --gcs_bucket TEXT             The name of the GCS bucket that holds
+                                OpenShift CI logs  [required]
+  --build_id TEXT               The build ID that needs to be reported. The
+                                value of $BUILD_ID
+  --job_name_safe TEXT          The safe name of a test in a Prow job. The
+                                value of $JOB_NAME_SAFE
+  --job_name TEXT               The full name of a Prow job. The value of
+                                $JOB_NAME
+```
+
+**Examples:**
+
+```commandline
+# Using all environment variables (Running in OpenShift CI uses this method)
+$ export BUILD_ID="some_build_id"
+$ export JOB_NAME_SAFE="some_safe_job_name"
+$ export JOB_NAME="some_job_name"
+$ export FIREWATCH_CONFIG="[{"step": "some-step-name","failure_type":"pod_failure", "classification": "some best guess classification", "jira_project": "PROJECT"}]"
+$ firewatch report
+
+# Using CLI arguments
+$ firewatch report --build_id some_build_id --job_name_safe some_safe_job_name --job_name some_job_name --firewatch_config_path /some/path/to/firewatch_config.json
+
+# Exit with a non-zero exit code if test failures are found in any JUnit file for a step
+$ firewatch report --fail_with_test_failures
+
+```
+
+**Example of Jira Ticket Created:**
+
+![Sreenshot of an example Jira ticket](images/jira-ticket-example.png)
```

Comment:
```suggestion
![Screenshot of an example Jira ticket](images/jira-ticket-example.png)
```
The screenshot does not show in the readme preview, though it appears in the compare window

![Pasted image](https://github.com/CSPI-QE/firewatch/assets/8645166/122c5fc7-9c5e-4668-a745-6c5adaf09d34)

Reply:
I think that may because the file hasn't been merged yet. I'll make the typo change and see if it starts working after the merge. Thanks for pointing that out!